sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -783,9 +783,6 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     struct ShareGetAccessPolicyResult
```

Comment:
```suggestion
    struct ShareAccessPolicy
```

Reply:
It's in details namespace, so it doesn't matter. We use typedef in `Models::` namespace. This is a limitation of Tank's generator.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/894da95ca1417fce9346168f7ca2cbf5f94503c5/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_responses.hpp#L40

Reply:
I see. Thanks for clarifying.